utils/loaddata.js

```diff
@@ -41,19 +41,19 @@ async function loadData(dataUrls) {
     for (let i = 0; i < files.length; i++) {
         const [url, csv_type] = files[i].split('|');
         if (url.trim() == '') continue
-        try{
+        try {
             await loadCSV(stream_client, url, csv_type)
         } catch(err) {
             stream_client.end()
```

Comment:
should we add:

` stream_client.on('end', () => manager.closeConnection(stream_client))` 

here too?

Reply:
Good catch-- moved it up so it's handled either way.